sections/sprint-5/limiting-transmissions.tex

```diff
@@ -1,11 +1,13 @@
 \section{Limiting the amount of UDP transmissions}\label{sec:sprint5-transmissionlimit}
 As the game was being tested we encountered a problem with the amount of UDP packages that were being transmitted.
 Initially there were no limitations to the number of packages sent, and the server would transmit as many as it possibly could.
-This would overburden the internet and could cause it to crash.
-A limit had to be set for how many packages were able to be transmitted per second.
-Since Pozyx can send updates on tags at a rate of approximately 60 per second, it would be preferable to have the limit at close to that amount \cite{pozyx-Performance}.
-The limit could even be a bit higher than 60, since the updates are being transmitted with UDP which means that there is no assurance that each package arrives.
-Eventually, a limit of 70 packages per second was chosen based on the Pozyx limit and the chance of packages not arriving.
+This would overburden the internet connection and could cause it to crash.
+To account for this, a simple form of \textit{Congestion Control} was implemented where a limit was set for how many packages were able to be transmitted per second.
+\textit{Congestion Control} is the act of modulating network traffic to prevent senders from overwhelming the network, which is typically done by reducing how many packets are sent \cite{CongestionControl}.
+Pozyx can send updates on tags at a rate of approximately 60 per second \cite{pozyx-Performance}.
+Since UDP packages are not guarenteed to arrive at the destination, the limit for transmissions needs to account for this.
+If a limit of 120 is set, every new update from pozyx will be transmitted twice, allowing the each new update to, in a sense, be retransmitted if it were to be lost.
+This is a form of \textit{Forward Error Correction} where redundancy is introduced to avoid the need for requesting retransmission of packets \cite{ForwardErrorCorrection}.
```

Comment:
```suggestion
This is a form of \textit{Forward Error Correction}, where redundancy is introduced to avoid the need for requesting retransmission of packets \cite{ForwardErrorCorrection}.
```